content/about.md

```diff
@@ -35,6 +35,12 @@ support issues.
 In September 2014, the project was renamed to "DB Browser for SQLite", to
 avoid confusion with an existing application called "Database Browser".
 
+Ten years later, the primary maintainers of DB Browser for SQLite are
```

Comment:
Good point.  We should add in more recent contributors.  @mgrojo, @chrisjlocke, @lucydodo all spring to mind easily.

@scottfurry was doing good stuff for a while too, but has moved on since.